src/utils/getApi.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios"
+
+export function getArticles () {
+    return axios
+    .get(`https://nc-news-1ldz.onrender.com/api/articles`)
+    .then((res) => {
+        return res.data
+    });
+}
```

Comment:
Good work, what if there's an error though? this api call could fail! have a `catch` block to deal with it if so